lib/auth/grpcserver.go

```diff
@@ -2508,6 +2508,14 @@ func validateUserSingleUseCertRequest(ctx context.Context, actx *grpcContext, re
 		return trace.BadParameter("unknown certificate Usage %q", req.Usage)
 	}
 
+	if req.RequesterName == proto.UserCertsRequest_TSH_DB_LOCAL_PROXY_TUNNEL &&
+		req.Usage == proto.UserCertsRequest_Database {
+		// don't limit the cert expiry to 1 minute for db local proxy tunnel,
+		// because the certs will be kept in-memory by the client to protect
+		// against cert/key exfiltration.
```

Comment:
Can we extend the commend and mention that In case of db local tunnel where the MFA is required the MFA certs expiration time is bound to the lifetime of the local proxy process ?